sapp/CMakeLists.txt

```diff
@@ -47,6 +47,27 @@ fips_begin_app(quad-sapp-ui windowed)
 fips_end_app()
 target_compile_definitions(quad-sapp-ui PRIVATE USE_DBG_UI)
 
+fips_ide_group(Samples)
+fips_begin_app(spritebatch-sapp windowed)
+    fips_vs_warning_level(3)
+    fips_files(spritebatch-sapp.c)
+    sokol_shader(spritebatch-sapp.glsl ${slang})
+    fips_deps(sokol)
```

Comment:
Please add the "stb" dependency here (and also remove the STB_IMAGE_IMPLEMENTATION macro. stb_image.h is already compiled into a library, and this also takes care of supressing warning in stb_image.h (which for instance has tons of sign-conversion warnings).

Reply:
Done